### ISSUE TITLE
Add commandline option to enhance pythonpath when starting the ride

### ISSUE THREAD

**Issue by robotframework-ride** [robotframework/RIDE#477]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=446) by bobobruhn on 4 May 2010*

Simmilar to pybot, the RIDE should support the options -P or --pythonpath
when started, to support appending pathes to the pythonpath at startup.

This is an advantage if you want to write testcases for different releases
that require other user testlibrary implementtation.

**Comment by robotframework-ride** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=446#c2) by @yanne on 11 Aug 2010*


You can always modify pythonpath via environment variables.

**Comment by robotframework-ride** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=446#c4) by samuli.silvius on 3 Nov 2010*


Yes, it's very annoying that after starting ride you notice that some of the libraries are not in pythonpath and keyword completion does not work. I always need to make starter script for ride where first set pythonpath and then start ride. Why I cannot load libraries through RIDE GUI by pointing wanted directories and press load etc. Then keyword completion would work.

**Comment by robotframework-ride** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=446#c5) by @pekkaklarck on 3 Nov 2010*


Samuli, please submit a separate issue about configuring RIDE's pythonpath via GUI. Once we create an UI for editing settings (that's needed pretty soon anyway) this will be pretty trivial.

**Comment by robotframework-ride** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=446#c6) by @pekkaklarck on 22 Sep 2011*


Samuli proposed adding a setting for configuring PYTHONPATH in comment 4. That is now <a class="closed_ref" href="https://code.google.com/p/robotframework-ride/issues/detail?id=856" title="Setting PYTHONPATH in RIDE"> issue 856 </a> and it will be included into the next release. I'm not sure is a separate command line option for this needed anymore after that. Most of the time having PYTHONPATH persistently stored works better, and in cases where you want to have custom PYTHONPATH for different projects (and having non-related items there actually causes problems) you can create a custom start-up script that sets PYTHONPATH environment variable before starting RIDE.

**Comment by robotframework-ride** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=446#c7) by @mkorpela on 10 Jun 2012*


In my opinion this issue has no value as we now have the setting and the settings editor.

Closing as WontFix.
